src/modules/bot/commands/questions/questions-interaction.collector.ts

```diff
@@ -155,12 +169,44 @@ export class QuestionsInteractionCollector {
       }, answerTimeInMilliseconds)
     }
 
+    this.stopButton.setDisabled(false)
```

Comment:
I suggest you remove this because user should be able to stop the bot's voice after click the listen button.